framework/include/materials/Material.h

```diff
@@ -231,7 +231,7 @@ class Material :
   std::set<std::string> _supplied_props;
 
   /// If False MOOSE does not compute this property
-  bool _compute;
+  const bool & _compute;
```

Comment:
This is fine but funny! 8 times the storage and slower to dereference! Not sure what you were going for here. 😄